interface/patient_file/report/custom_report.php

```diff
@@ -837,7 +844,19 @@ function postToGet($arin)
 
     if ($PDF_OUTPUT == 1) {
         try {
-            $pdf->Output($fn, $GLOBALS['pdf_output']); // D = Download, I = Inline
+            if ($PDF_FAX === 1) {
+                $fax_pdf = $pdf->Output($fn, 'S');
+                $tmp_path = $GLOBALS['OE_SITE_DIR'] . '/messageStore/send/';
```

Comment:
Writable directories need to be in documents tree and directory names should not have caps (unless in the class tree).

Reply:
I need a writable directory from outside openemr for fax file dumps. Since when has default not been writable?
It's my directory so i'll name how I want:) no, i'll refactor

Reply:
It's a universal rule:
```
Thou shall not kill
Thou shall only write in documents tree
```
:)

Reply:
Here's the PR that folded everything into documents folder:
https://github.com/openemr/openemr/pull/2254

And check out Item 2 and Item 3 (they are striked out) here:
https://github.com/openemr/openemr/issues/2181

By doing this, we leverage apache securing of the documents folder to all other subfolder. If you place it in just sites/default/another directory, then a user can see the file without even logging into openemr by simply typing name of it in apache web address. Note we did add on another layer of security in case the documents directory does get exposed (bad apache setting for example) via the encryption (goal is to encrypyt enything that gets saved there even if only temporary).